chart/Chart.yaml

```diff
@@ -3,19 +3,19 @@ name: tobs
 description: A Helm chart for tobs, The Observability Stack for Kubernetes
 home: https://github.com/timescale/tobs
 sources:
-- https://github.com/timescale/tobs
+  - https://github.com/timescale/tobs
 
 maintainers:
-- name: timescale
-  url: https://www.timescale.com/
+  - name: timescale
+    url: https://www.timescale.com/
 
 keywords:
-- observability
-- monitoring
-- tracing
-- opentelemetry
+  - observability
+  - monitoring
+  - tracing
+  - opentelemetry
 
-version: 14.0.0
+version: 14.1.1
```

Comment:
```suggestion
version: 14.0.1
```

Reply:
well I assumed it would be merged after #564.  I don't think that will happen now and can update this